spec/services/submission/plant_trial_exporter_spec.rb

```diff
@@ -21,40 +26,32 @@
 
       documents = subject.documents
 
-      expect(documents.size).to eq 5
+      expect(documents.size).to eq 3
       expect(documents[:plant_trial].lines.size).to eq 2
       expect(documents[:plant_trial].lines[1].split(',')[0]).
         to eq plant_trial.plant_trial_name
 
-      expect(documents[:plant_scoring_units].lines.size).to eq 3
-      expect(documents[:plant_scoring_units].lines[1,2].map{ |l| l.split(',')[0] }).
-        to match_array psus.map(&:scoring_unit_name)
-
-      expect(documents[:plant_accessions].lines.size).to eq 3
-      expect(documents[:plant_accessions].lines[1,2].map{ |l| l.split(',')[0] }).
-        to match_array psus.map{ |psu| psu.plant_accession.plant_accession }
-      expect(documents[:plant_accessions].lines[1,2].map{ |l| l.split(',')[2] }).
-        to match_array [
-          psus[0].plant_accession.plant_line.plant_variety.plant_variety_name,
-          psus[1].plant_accession.plant_variety.plant_variety_name
-        ]
-
       expect(documents[:trait_descriptors].lines.size).to eq 3
       expect(documents[:trait_descriptors].lines[1,2].map{ |l| l.split(',')[1] }).
         to match_array tds.map(&:trait_name)
 
-      expect(documents[:trait_scoring].lines.size).to eq 3
-      expect(documents[:trait_scoring].lines[0].strip.split(',')).
-        to eq ['Scoring unit name', "#{tds[0].trait_name} rep1", "#{tds[0].trait_name} rep2", tds[1].trait_name]
+      expect(documents[:trait_scoring].lines.size).to eq 4
+      expect(documents[:trait_scoring].lines[0].strip.split(',')[13,3]).
+        to eq ["#{tds[0].trait_name} rep1", "#{tds[0].trait_name} rep2", tds[1].trait_name]
 
-      expect(documents[:trait_scoring].lines[1,2].map{ |l| l.split(',')[0] }).
+      generated_scores = CSV.parse(documents[:trait_scoring]).map{ |row| row[13,3] }[1,3]
+      expect(documents[:trait_scoring].lines[1,3].map{ |l| l.split(',')[0] }).
         to match_array psus.map(&:scoring_unit_name)
-      expect(documents[:trait_scoring].lines[1,2].map{ |l| l.split(',')[1] }).
-        to match_array [ts1.score_value, '-']
-      expect(documents[:trait_scoring].lines[1,2].map{ |l| l.split(',')[2] }).
-        to match_array [ts2.score_value, '-']
-      expect(documents[:trait_scoring].lines[1,2].map{ |l| l.strip.split(',')[3] }).
-        to match_array [ts3.score_value, '-']
+      expect(generated_scores.map{ |l| l[0] }).to match_array [ts1.score_value, '-', '-']
```

Comment:
I got the impression that order of scores should be important here, shouldn't it?